Source/LDSwiftEventSource.swift

```diff
@@ -95,17 +95,24 @@ class EventSourceDelegate: NSObject, URLSessionDataDelegate {
 
     private let delegateQueue: DispatchQueue = DispatchQueue(label: "ESDelegateQueue")
 
-    private var readyState: ReadyState = .raw
+    private var readyState: ReadyState = .raw {
+        didSet {
+            #if !os(Linux)
+            os_log("State: %@ -> %@", log: logger, type: .debug, oldValue.rawValue, readyState.rawValue)
+            #endif
+        }
+    }
```

Comment:
Previously we only sometimes logged changes to this variable, so I just added logging to a property observer instead.